code/modules/vehicles/mecha/combat/gygax.dm

```diff
@@ -13,7 +13,6 @@
 	destruction_sleep_duration = 40
 	exit_delay = 40
 	encumbrance_gap = 1.4
-	internal_damage_threshold = 18
```

Comment:
comments for removal

Reply:
uuuh it's a removal of a edit previously on my previous pr but ok